From f1bd999df30c6e88c1a967856304086aa72f2db0 Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Mon, 11 Jan 2016 10:23:55 +0800
Subject: [PATCH] MLK-12172: ARM: dts: imx7d-sdb-m4: disable reg_can2_3v3

Disable can2 is not enough, since the reg_can2_3v3 will be turned off by
the regulator framwork and that will impact can2 in m4 side even if can2
driver disabled in A7 side.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
---
 arch/arm/boot/dts/imx7d-sdb-m4.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb-m4.dtsi b/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
index 021a874..59bc1dc 100644
--- a/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
+++ b/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
@@ -54,6 +54,10 @@
 	reg = <0x00901000 0xf000>;
 };
 
+&reg_can2_3v3 {
+	status = "disabled";
+};
+
 &rpmsg{
 	status = "okay";
 };
-- 
1.8.0

